app/templates/components/public/ticket-list.hbs

```diff
@@ -80,25 +80,29 @@
               {{/if}}
             </td>
             <td>
-              <div class="field">
-                <UiDropdown
-                  @class="compact selection"
-                  @forceSelection={{false}}
-                  @onChange={{action "updateOrder" ticket}} as |execute mapper|>
-                  <Input
-                    @type="hidden"
-                    @id={{concat ticket.id "_quantity"}}
-                    @value={{ticket.orderQuantity}} />
-                  <i class="dropdown icon"></i>
-                  <div class="default text">{{if (eq this.tickets.length 1) (max ticket.minOrder 1) 0}}</div>
-                  <div class="menu">
-                    <div class="item" data-value="{{map-value mapper 0}}">0</div>
-                    {{#each (range (max ticket.minOrder 1) (add ticket.maxOrder 1)) as |count|}}
-                      <div class="item" data-value="{{map-value mapper count}}">{{count}}</div>
-                    {{/each}}
-                  </div>
-                </UiDropdown>
-              </div>
+              {{#if (sub ticket.quantity (add ticket.orderStatistics.tickets.placed ticket.orderStatistics.tickets.completed))}}
```

Comment:
.

Reply:
Ok understood, waiting for @SARTHAK1SINGH to follow up else I'll complete it